oak/server/rust/oak_runtime/src/io.rs

```diff
@@ -89,6 +89,9 @@ impl<T: Decodable> Receiver<T> {
                     })
                 })
                 .and_then(|message| T::decode(&message))
+        } else if read_status[0] == ChannelReadStatus::Orphaned {
+            info!("Channel closed {:?}", self.handle);
```

Comment:
Is it an intended behavior? Should it be a `warn!(...)`?

Reply:
The t_i_r node creates ephemeral gRPC client pseudo-Nodes, and when they're `drop`ped the invocation channel is auto-closed and this pops out here, so I *think* it should be an `info`.

To put it another way: if this should be a `warn`, then `get_database_entry()` in `t_i_r` should probably be changed to use a persistent client.